frontend/packages/console-shared/src/components/spotlight/Spotlight.scss

```diff
@@ -0,0 +1,64 @@
+@keyframes expand {
```

Comment:
```suggestion
@keyframes co-spotlight-expand {
```

Reply:
done

Reply:
Shouldn't this be `ocs-spotlight-expand`?

Reply:
yes it should be. my mistake

Reply:
renamed to `ocs-spotlight-expand`